### PR TITLE
Rendering link to register register on home page

### PR DESCRIPTION
**Pull request by om-sharma**
Changes renders a link on home page to an entry in register register.

### CODE REVIEW

File: src/main/resources/assets/css/main.css [hunk 1]
```diff
@@ -1045,3 +1045,6 @@ table {
     background-color: #f5f5f5; }
   table.entry th, table.entry td {
     border-bottom: 1px solid #bfc1c3; }
+
+.no-underline a {
+  text-decoration: none; }
```
Comment (philandstuff) [inline src/main/resources/assets/css/main.css]: this is a stray update from somewhere else?

File: src/main/resources/templates/home.html [hunk 2]
```diff
@@ -27,6 +27,8 @@ <h2 class="heading-large">Technical information</h2>
             <p>Registers are made up of entries containing fields of data. To build services on top of these registers you can access the data in a variety of formats.</p>
             <h3>Fields in this register:</h3>
             <div class="field-list"> <ul><li th:each="field : ${register.fields}" th:text="${field}"></li></ul> </div>
+
+            <h4 class="no-underline"><a th:href="${linkToRegisterRegister}" th:text="${'Meta information about ' + registerId + ' register'}"></a></h4>
```
Comment (philandstuff) [inline src/main/resources/templates/home.html:31]: This is not a heading.

File: src/main/sass/main.scss [hunk 3]
```diff
@@ -169,3 +169,8 @@ table {
   }
 }
 
+//link
+
+.no-underline a {
+    text-decoration: none;
```
Comment (philandstuff) [inline src/main/sass/main.scss:175]: why did you add this class? Can we have a consistent approach to link styling? eg I've started adding `a:hover { text-decoration: underline; }` but this link will not have this behaviour.

**Comment by om-sharma** [comment]
Closing PR. Waiting for visuals from Stephen and then will reopen.
